src/components/FlexibleForm/ControlDecorators/withRepeatableActions/withRepeatableActions.js

```diff
@@ -39,11 +39,21 @@ export const withRepeatableActions = memo(props => {
   }));
 
   const needsTranslation = wrapperLabel && !isFormattedMessage(wrapperLabel) && isTranslationId(wrapperLabel);
-  const hasRecords = !isEmpty(referenceTable);
+
+  const legendHeadline = (
+    <Headline
+      tag="h3"
+      margin="xx-small"
+    >
+      <FormattedMessage id={legend} />
+    </Headline>
+  );
 
   return (
-    <div className={classNames(styles.decorator, isEmpty(legend) ? styles['no-legend'] : '')}>
-      {enabled && hasRecords && (
+    <div className={styles.decorator}>
+      {legend && legendHeadline}
+
```

Comment:
Remove blank line, please

Reply:
why? It's two different components and in this way easy to read code, as for me?

Reply:
In my opinion, it doesn't make code more readable here, e.g. you don't put an empty line between tags or components, right? But it's up to you.

Reply:
ok, will remove. Just want to understand the code style rules on the project. Thanks!

Reply:
Guys, it is up to you to decide on such things, and once it is established then be consistent. As I remember since the beginning of the data-import project I also followed the rule of avoiding empty lines in such cases. I think a document can be added to the project for code standards like this.